components/dropdown/dropdown-content-styles.js

```diff
@@ -0,0 +1,148 @@
+import { css } from 'lit-element/lit-element.js';
+
+export const dropdownContentStyles = css`
+	:host {
+		box-sizing: border-box;
+		color: var(--d2l-color-ferrite);
+		display: none;
+		left: 0;
+		position: absolute;
+		text-align: left;
+		top: calc(100% + var(--d2l-dropdown-verticaloffset, 20px));
+		width: 100%;
+		z-index: 1000; /* position on top of floating buttons */
+	}
+
+	:host([opened]) {
+		display: inline-block;
+	}
+
+	:host([opened]) {
+		-webkit-animation: d2l-dropdown-animation 300ms ease;
+		animation: d2l-dropdown-animation 300ms ease;
+	}
+
+	:host([opened-above]) {
+		bottom: calc(100% + var(--d2l-dropdown-verticaloffset, 20px));
+		top: auto;
+		-webkit-animation: d2l-dropdown-above-animation 300ms ease;
+		animation: d2l-dropdown-above-animation 300ms ease;
+	}
+
+	:host .d2l-dropdown-content-pointer {

```

Comment:
The plain `:host` prefixes shouldn't be needed -- the style is already scoped to inside the host by shadow DOM.